evergreen.yml

```diff
@@ -907,23 +907,45 @@ functions:
           # Confirm semgrep version
           semgrep --version
           set +e
-          timestamp=$(date +'%Y-%m-%dT%H_%M_%S')
-          semgrep --config p/rust --metrics=off --sarif --exclude "benchmark" --verbose --error --severity=ERROR --sarif-output=mongosqltranslate_$timestamp.sast.sarif > mongosqltranslate_$timestamp.sast.cmd.verbose.out 2>&1
+          semgrep --config p/rust --metrics=off --sarif --exclude "benchmark" --verbose --error --severity=ERROR --sarif-output=mongosql.sast.sarif > mongosql.sast.cmd.verbose.out 2>&1
```

Comment:
We have this output file, I don't see it being uploaded or displayed.  We can `cat` it on failure to see what's going on. 
```
  if [ $SCAN_RESULT -ne 0 ]; then
    echo "semgrep failed with exit code: $SCAN_RESULT."
    cat mongosql.sast.cmd.verbose.out
  fi
  ```

Reply:
@bucaojit I made changes to copy the files as part of a post function. This is similar to what is done in JDBC to copy test results for example.
Here is an example run of when there is an issue found by the analysis : https://spruce.mongodb.com/version/67eeab567c467900073cb499/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC
It reports the task as failing and copies the files.
And it also copies the file when things work.

Reply:
Okay that looks good 👍